static/stable/prod/modules/fed-modules.json

```diff
@@ -199,6 +199,9 @@
             }
         ]
     },
+    "costManagementMfe": {
+        "manifestLocation": "/apps/cost-management-mfe/fed-mods.json"
```

Comment:
This will require an empty modules array. Chrome currently expects the array to exist. We will address it on the UI side, but don't want to block your release.
```
modules: []
```

Reply:
Added the empty modules array